tests/extraction_tests/resources/mock_sheet_data/funding_profiles_mock.csv

```diff
@@ -19,7 +19,7 @@ All the totals are automatically calculated.  If you need a table with more rows
 
 ",,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,
```

Comment:
I think if we go with this approach of have different mock data sources for HS and FHSF, we should at least split them into different sub-folders to keep it clearer.

Reply:
same for assertion data

Reply:
I have now changed mock data to a single source of truth